src/carousel.css.ts

```diff
@@ -170,26 +172,26 @@ export const carouselItem = style({
     },
     '@media': {
         [mq.tabletOrSmaller]: {
```

Comment:
I think this should be `mq.tablet`, to avoid overlapping with `mq.mobile`

Reply:
this comment is obsolete as the solution changed ;)